awx/api/views/__init__.py

```diff
@@ -3013,6 +3012,33 @@ def is_valid_relation(self, parent, sub, created=False):
         return None
 
 
+class WorkflowJobTemplateNodeCreateApproval(RetrieveAPIView):
```

Comment:
I still have trouble with consistency of terminology. This creates an "approval template" or maybe an "approval job template", or a "workflow approval template". The template spawns a "workflow approval" or an "approval job" or an "approval".

By whatever convention, this view name is confusing, since it create a template-like object.